src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -1150,56 +1150,18 @@ def forward(
         return (pred_logits, image_class_embeds)
 
 
-class OwlViTImageTextEmbedder(nn.Module):
-    def __init__(self, config: OwlViTConfig):
-        super().__init__()
-
-        self.clip = OwlViTModel(config)
-        self.layer_norm = nn.LayerNorm(config.vision_config.hidden_size)
-
-    def forward(
-        self,
-        pixel_values: Optional[torch.FloatTensor] = None,
-        input_ids: Optional[torch.Tensor] = None,
-        attention_mask: Optional[torch.Tensor] = None,
-        output_attentions: Optional[bool] = None,
-    ) -> Tuple[torch.FloatTensor, torch.FloatTensor]:
-
-        image_embeds, text_embeds = None, None
-
-        # Encode text
-        if input_ids is not None:
-            text_embeds = self.clip.get_text_features(
-                input_ids=input_ids, attention_mask=attention_mask, output_attentions=output_attentions
-            )
-
-        # Encode image
-        if pixel_values is not None:
-            image_embeds = self.clip.get_image_features(
-                pixel_values, return_projected=False, output_attentions=output_attentions
-            )
-
-            # Resize class token
-            new_size = tuple(np.array(image_embeds.shape) - np.array((0, 1, 0)))
-            class_token_out = torch.broadcast_to(image_embeds[:, :1, :], new_size)
-
-            # Merge image embedding with class tokens
-            image_embeds = image_embeds[:, 1:, :] * class_token_out
-            image_embeds = self.layer_norm(image_embeds)
-
-        return (image_embeds, text_embeds)
-
-
 class OwlViTForObjectDetection(OwlViTPreTrainedModel):
     config_class = OwlViTConfig
     main_input_name = "pixel_values"
 
     def __init__(self, config: OwlViTConfig):
         super().__init__(config)
 
-        self.embedder = OwlViTImageTextEmbedder(config)
+        self.owlvit = OwlViTModel(config)
```

Comment:
Make sure to update the `base_model_prefix` to "owlvit"

Reply:
Yes, it is set to "owlvit"